src/pages/version/Tasks.tsx

```diff
@@ -131,9 +131,14 @@ const getQueryVariables = (
     [PatchTasksQueryParams.Sorts]: sorts,
   } = parseQueryString(search);
 
+  // Don't include duration when sorting on PatchTasks table.
+  const filteredSorts = parseSortString(sorts).filter(
+    (sort) => sort.Key !== "DURATION"
+  );
```

Comment:
This doesn't seem necessary? maybe the two tabs should handle their own sorting and query param logic?